### PR TITLE
Adding egress IP addresses to the CRD output

### PR DESCRIPTION
**Pull request by alexanderConstantinescu**
This addresses bug [1707749](https://bugzilla.redhat.com/show_bug.cgi?id=1707749)

Please let me know if the description should be modified.

/Alex

### CODE REVIEW

**Comment by danwinship** [comment]
What output does this result in? I thought I had tried this before and determined that the CRD printer didn't support array-valued columns.

**Comment by alexanderConstantinescu** [comment]
> What output does this result in? I thought I had tried this before and determined that the CRD printer didn't support array-valued columns.

Please find an attached screenshot of a local edit with bogus IPs. 

`NAME                                                    NETID      EGRESSIPS
default                                                 0          
kube-node-lease                                         14579022   
kube-public                                             9731826    
openshift-sdn                                           15140382   [89.16.153.196 98.65.165.22]
`

File: bindata/network/openshift-sdn/001-crd.yaml [hunk 1]
```diff
@@ -117,6 +117,10 @@ spec:
     type: string
     description: The CIDR range of the overlay network assigned to the node for its pods
     JSONPath: .subnet
+  - name: EgressIPs
```
Comment (danwinship) [inline bindata/network/openshift-sdn/001-crd.yaml]: In HostSubnet, you should also print the `EgressCIDRs`, before the `EgressIPs`. (https://github.com/openshift/origin/blob/release-3.11/pkg/printers/internalversion/printer.go#L66)

Comment (alexanderConstantinescu) [reply bindata/network/openshift-sdn/001-crd.yaml]: PR updated

**Pull request merged**